rewrite-yaml/src/test/kotlin/org/openrewrite/yaml/ChangePropertyKeyTest.kt

```diff
@@ -122,14 +125,16 @@ class ChangePropertyKeyTest : YamlRecipeTest {
     @Test
     @Issue("https://github.com/openrewrite/rewrite/issues/1114")
     fun `change path to one path longer`() = assertChanged(
-        recipe = ChangePropertyKey("a.b.c", "a.b.c.d", null, null),
+        recipe = ChangePropertyKey("a.b.c", "a.b.c.d", null, null, null),
         before = "a.b.c: true",
-        after = "a.b.c.d: true"
+        after = "a.b.c.d: true",
+        cycles = 1,
+        expectedCyclesThatMakeChanges = 1
```

Comment:
Recipes must not depend on being run for exactly one cycle for correctness. Some other recipe may trigger another cycle. Users may run the recipe twice. 

Recipes cannot depend on being run exactly once.

Reply:
Fixed in https://github.com/openrewrite/rewrite/pull/2066/commits/90b7d73ae5004023abd61a9f3041168f48625557